src/line-chart/line-chart.component.ts

```diff
@@ -175,7 +178,6 @@ export class LineChartComponent extends BaseChartComponent {
   @Input() yAxisTickFormatting: any;
   @Input() roundDomains: boolean = false;
   @Input() tooltipDisabled: boolean = false;
-  @Input() showSeriesOnHover: boolean = true;
```

Comment:
thanks for explanation.  I understand it is out for performance improvement, but sometime it is important to show dots on line series. I wonder if there is a plan to address that. (Display all dots on the lines).